app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
Hmmm. Maybe we are talking cross-purposes. If so then my apologies. I've seen no mention of `optional.js` in the docs. To me this was just an "indicator" that one could have more than one reducer file (something that is _very_ common in any real app).

If we were to have it though it would be better to include it as part of the blueprinting system so that you're supporting both Apps and Addons. Or am I missing something?